file-formats/gsmg/type/zif_aff_gsmg_v1.intf.abap

```diff
@@ -0,0 +1,92 @@
+INTERFACE zif_aff_gsmg_v1
+  PUBLIC.
+
+  TYPES:
+
+    "! <p class="shorttext">Category</p>
+    "! Category
+    "! $values {@link zif_aff_gsmg_v1.data:co_category}
+    "! $default {@link zif_aff_gsmg_v1.data:co_category.usage}
+    ty_category             TYPE c LENGTH 10,
+
+    "! <p class="shorttext">Detailed Description</p>
+    "! Detailed description
+    ty_detailed_description TYPE c LENGTH 255,
+
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Category</p>
+      "! Category
+      "! $required
+      category             TYPE ty_category,
+
+      "! <p class="shorttext">Detailed Description</p>
+      "! Detailed description
+      "! $required
+      detailed_description TYPE ty_detailed_description,
+
+    END OF ty_general_information.
+
+
+  CONSTANTS:
+
+    "! <p class="shorttext">Category</p>
+    "! Category
+    BEGIN OF co_category,
+
+      "! <p class="shorttext">Availability</p>
+      "! Availability
+      availability    TYPE ty_category VALUE 'AVAIL',
+
+      "! <p class="shorttext">Performance</p>
+      "! Performance
+      performance     TYPE ty_category VALUE 'PERFORM',
+
+      "! <p class="shorttext">Exceptions</p>
+      "! Exceptions
+      exceptions      TYPE ty_category VALUE 'EXCEPTION',
```

Comment:
Valid point. Plural makes no sense for most of the category constants. But it would make sense for 'USAGE' and this is singular as well. So I'll change it to 'EXCEPTION'.